packages/flutter/lib/src/material/checkbox_list_tile.dart

```diff
@@ -356,6 +357,11 @@ class CheckboxListTile extends StatelessWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
+  /// Defines insets for [Checkbox], [title], [subtitle], and [secondary].
```

Comment:
Just a suggestion for improvement to the wording, but you can do something else that might make more sense to you. I'm trying to use wording to distinguish between the insets for the entire tile vs. padding that insets every single widget in the tile (ie. 10px padding surrounding the tile's entire contents vs 10px padding surrounding checkbox, 10px padding surrounding title, etc).
```suggestion
  /// Defines insets surrounding the tile's contents. 
  ///
  /// This value will surround the [Checkbox], [title], [subtitle], and [secondary]
  /// widgets in [CheckboxListTile].
```